Gemfile

```diff
@@ -139,7 +132,6 @@ group :offline do
   gem 'rdoc'
   gem 'rubocop'
   gem 'rubocop-git'
-  gem 'thin'
```

Comment:
Hmm I checked and it's not necessary for running Markus. It might have been used in the pass to serve HTML documentation files generated by rdoc, but I guess that hasn't been used in a while.

Reply:
Rubymine runs it as the default web server when it's in the gemfile. What is the alternative in dev mode? Using unicorn?

Reply:
Rails comes bundled with [Webrick](https://en.wikibooks.org/wiki/Ruby_on_Rails/Getting_Started/Running_the_Server), which is okay for development.